.travis.yml

```diff
@@ -8,8 +8,9 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+
     - CONDA_PY=27
+    - CONDA_PY=34
```

Comment:
Are you familiar with [conda-smithy](https://github.com/conda-forge/staged-recipes/wiki/conda-smithy-rerender)?

I recommend you to re-render using that instead of changing the feedstock boilerplate configuration manually.